### DISCUSSION TITLE
Extend the SubView's sortField attribute to allow sorting by multiple fields

### DISCUSSION THREAD

**Discussion by maxpatiiuk** [specify/specify7#5574] [Feature Requests]
Like `isCurrent,-timestampCreated`

Description of the sortField attribute: https://github.com/specify/specify7/wiki/Form-System#:~:text=sortField%20(string%2C%20optional%2C%20case%2Dinsensitive)

**Comment by specifysoftware** [comment]
This issue has been mentioned on **Specify Community Forum**. There might be relevant details there:

https://discourse.specifysoftware.org/t/order-subviews-by-multiple-related-fields/2155/1

**Comment by bronwyncombs** [comment]
Philippe @ IRD add support for this on [discourse](https://discourse.specifysoftware.org/t/order-subviews-by-multiple-related-fields/2155)

>For some loans we may have dozens or even hundreds of loan preparations, so my colleagues asked for sorting default grid view by Collectors, and Collector number.
>
>Since Specify 7.7 I read that [subviews accept one sort field argument](https://discourse.specifysoftware.org/t/order-subviews-by-field/371) sortField=${fieldName}.
>
>This is how I’d like to initialize the subview:
`<cell type="subview" viewname="LoanItems_Aublet3_Form" id="10" name="loanPreparations" defaulttype="table" initialize="sortField=preparation.collectionObject.collectingevent.collectors,preparation.collectionObject.collectingevent.stationFieldNumber" colspan="17" rows="4"/>`
